pages/Identity/Factory.md

```diff
@@ -75,26 +75,33 @@ Clones follow [EIP 1167](https://eips.ethereum.org/EIPS/eip-1167) for a **non-up
  * @dev DSNP Identity Factory Interface for creating identities via [EIP 1167](https://eips.ethereum.org/EIPS/eip-1167)
  */
 interface IIdentityCloneFactory {
+
+    /**
+     * @dev event to log the created proxy contract address
+     */
+    event ProxyCreated(address addr);
+
     /**
      * @dev Creates a new identity with the message sender as the owner
      * @dev [EIP 1167](https://eips.ethereum.org/EIPS/eip-1167) Proxy 
      * @param logic The Logic address to use for identity creation
      * 
-     * @returns The address of the newly created Identity
+     * @return The address of the newly created Identity
      */
     function createClone(address logic) public returns (address);
 
     /**
      * @dev Creates a new identity with the ecrecover address as the owner
      * @dev [EIP 1167](https://eips.ethereum.org/EIPS/eip-1167) Proxy
+     * @param v EIP-155 calculated Signature v value
      * @param r ECDSA Signature r value
      * @param s ECDSA Signature s value
-     * @param v EIP-155 calculated Signature v value
      * @param logic The Logic address to use for identity creation
      * 
-     * @returns The address of the newly created Identity
+     * @dev This MUST emit ProxyCreated with the address of the new proxy contract
+     * @return The address of the newly created identity proxy contract
```

Comment:
Is this still necessary? Seems like an added expense.

Reply:
It is an extra 236 gas, but it is needed in the case of other contracts calling it.